app/views/favourite_rooms/list.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2 class="mb-3 fw-light"><%= 'These are your favourite rooms:' %></h2>
+
+<% if current_user %>
+    <% @favourite_rooms.each do |favourite_room| %>
+        <ul>
+            <li>
+                <%= link_to favourite_room.room.name, room_path(favourite_room.room) %>
+                <td class="td-btn">
+                <button class="delbttn btn btn-secondary" data-room-id="<%= favourite_room.room.id %>" title="<%= t('unfavourite') %>">
+                    <i class="fas fa-trash"></i>
+                </button>
+                </td>
+            </li>
+        </ul>
+    <% end %>
+<% else %>
+    <p> Login to see your favourite list. </p>
+<% end %>

```

Comment:
Good job! However, I think there is now the `/error/show` route for these kinds of error? This is at least the page I get redirected to for other permission denied pages.

Reply:
According to RH team not the way to go. Added some CSS